modules/core/test/test_intrin.avx2.cpp

```diff
@@ -0,0 +1,43 @@
+// This file is part of OpenCV project.
+// It is subject to the license terms in the LICENSE file found in the top-level directory
+// of this distribution and at http://opencv.org/license.html.
+#include "test_precomp.hpp"
+#include "test_intrin.simd.hpp"
+
+namespace opencv_test { namespace hal {
+using namespace CV_CPU_OPTIMIZATION_NAMESPACE;
+
+TEST(hal_intrin256, uint8x32)
```

Comment:
It is "dead" test for non-AVX2 machines - any code from this file should not be called there.

Test declaration (via `TEST()`) and dispatching should be done outside of this function. See "FP16" test:
```
    CV_CPU_CALL_FP16_(test_hal_intrin_float16, ());
    throw SkipTestException("Unsupported hardware: FP16 is not available");
```

Reply:
Done, thank you !

Reply:
Accidentally rebase removed the last changes, I think its fine now .. thank you once again